xrpl/asyncio/transaction/reliable_submission.py

```diff
@@ -107,3 +114,90 @@ async def send_reliable_submission(
     return await _wait_for_final_transaction_outcome(
         transaction_hash, client, prelim_result, transaction.last_ledger_sequence
     )
+
+
+async def _get_signed_tx(
+    transaction: Union[Transaction, str],
+    client: Client,
+    wallet: Optional[Wallet] = None,
+    check_fee: bool = True,
+    autofill: bool = True,
+) -> Transaction:
+    """
+    Initializes a signed transaction for a submit request.
```

Comment:
Initializes is a bit vague - attempted to add more of the keywords/nuance.

```suggestion
    Sets up a transaction to be submitted by optionally autofilling, signing, and validating the fee seems reasonable.
```